docker-compose.yml

```diff
@@ -38,6 +38,20 @@ services:
     env_file: .env
     container_name: mysql
 
+  postgres:
+    image: postgres:9.6.3
+    environment:
+      - POSTGRES_DB=${DB_NAME}
+      - POSTGRES_USER=${DB_USER}
+      - POSTGRES_PASSWORD=${DB_PASSWORD}
+    volumes:
+      - ./docker:/app/docker
+      - ./docker/psql:/etc/psql/conf.d
```

Comment:
Hmm not sure. Its probably not necessary for now until we do something with it in the future.